packages/cli/src/bin/ceramic.ts

```diff
@@ -49,7 +49,7 @@ program
   )
   .option(
     '--cors-allowed-origins <list>',
-    'Space-separated list of strings and/or regex expressions to set for Access-Control-Allow-Origin . Defaults to all: "*"'
+    'Space-separated list of strings and/or regex expressions to set for Access-Control-Allow-Origin . Defaults to all: ".*"'
```

Comment:
```suggestion
    'Space-separated list of strings and/or regex expressions to set for Access-Control-Allow-Origin . Defaults to all: "/.*/"'
```

Reply:
hmm... interesting. In my testing, if I specify `".*"` as the input string, then when it's parsed it winds up as a RegExp that displays as `"/.*/"`.  If I pass `"/.*/"` as input then it shows up as `"\\/.*\\/"` when parsed

Reply:
> hmm... interesting. In my testing, if I specify `".*"` as the input string, then when it's parsed it winds up as a RegExp that displays as `"/.*/"`. If I pass `"/.*/"` as input then it shows up as `"\\/.*\\/"` when parsed

Yes you are correct we don't need the slashes because it is a string input that gets converted to regex.

Reply:
okay great, then I'll leave it as-is.  Thanks!